infrastructure/helm/quantumserverless/README.md

```diff
@@ -12,7 +12,38 @@ helm repo add kuberay https://ray-project.github.io/kuberay-helm
 ```shell
 helm dependency build
 ```
+Update values.yaml file. Find and replace the following strings
 
+- **CLIENTSECRET-CHANGEME**: string used as the secret for a OIDC protocol
+- **HELM-RELEASE**: release name used in the helm install command
+- **LOCAL-IP**: IP address that can be accessed from both outside of the cluster and inside of the cluster.  
+
+**LOCAL-IP Example**
+
+MacOS - ifconfig output (**192.168.4.23**)
+```
+en0: flags=8963<UP,BROADCAST,SMART,RUNNING,PROMISC,SIMPLEX,MULTICAST> mtu 1500
+	options=6463<RXCSUM,TXCSUM,TSO4,TSO6,CHANNEL_IO,PARTIAL_CSUM,ZEROINVERT_CSUM>
+	ether a4:83:e7:27:70:71
+	inet6 fe80::8b4:58c9:11dd:e7e0%en0 prefixlen 64 secured scopeid 0x6
+	inet 192.168.4.23 netmask 0xfffffc00 broadcast 192.168.7.255
+	nd6 options=201<PERFORMNUD,DAD>
+	media: autoselect
+	status: active
+```
+Ubuntu - ifconfig output (**169.62.189.94**)
```

Comment:
fwiw, I am not able to find an IP address that works on Fedora using a kind cluster, so I'm getting these kinds of errors on the kuberay head pod:
```console
$ k logs -n ray quantum-serverless-kuberay-head-tswxj -c oauth-proxy
[2023/02/28 17:37:16] [provider.go:55] Performing OIDC Discovery...
[2023/02/28 17:37:16] [main.go:60] ERROR: Failed to initialise OAuth2 Proxy: error intiailising provider: could not
 create provider data: error building OIDC ProviderVerifier: could not get verifier builder: error while discovery 
OIDC configuration: failed to discover OIDC configuration: error performing request: Get 
"http://192.168.123.1:31059/realms/quantumserverless/.well-known/openid-configuration": dial tcp 
192.168.123.1:31059: connect: connection refused
```